lib/postprocessing/image.js

```diff
@@ -30,6 +30,14 @@ async function read(infile) {
 }
 
 async function applyOperations(img, instructions) {
+    // handle exif orientation
+    // MUST happen before img.resize() for orientation 5 to 8 (otherwise width & height must be swapped)
+    // not using img.autoOrient() because that does -strip which strips the entire exif metadata
+    // but we only want to remove the exif Orientation flag which we simply unset
+    img.out("-auto-orient");
+    img.out("-orient");
+    img.out("Undefined");
```

Comment:
What is that for?

Reply:
See the comment above. `-orient Undefined` unsets/removes the orientation flag from the metadata.

Reply:
Each argument must be separate with `out()`. `img.out("-orient Undefined")` does not work.